src/components/Navbar.tsx

```
@@ -1,22 +1,38 @@
+                    <h3 className={styles["title"]}>Old School Game</h3>
+                </Link>
+
+                <button className={styles["menu-icon"]} onClick={toggleMenu} aria-label="Toggle menu">
+                    {menuOpen ? <FaTimes /> : <FaBars />}
+                </button>
+
+                <div className={`${styles["nav-links"]} ${menuOpen ? styles["active"] : ""}`}>
+                    <Link href="/" className={styles["nav-link"]} onClick={closeMenu}>Home</Link>
+                    <Link href="/game" className={styles["nav-link"]} onClick={closeMenu}>Games</Link>
+                    <Link href="https://github.com/ajaynegi45/Old-School-Game" target="_blank" className={styles["nav-link"]} onClick={closeMenu}>GitHub</Link>
+                    <button className={styles["icon-button"]} aria-label="User profile">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)